addons/base_vat/models/res_partner.py

```diff
@@ -59,6 +59,14 @@
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
+    vat_validation_type = fields.Selection(selection_add=[('european_vat', 'European Vat')])
```

Comment:
What about use the `partner.country_id` in order to avoid register to times a similar field (A field for country `México` and a field for vat_type `mx`)?